### PR TITLE
update of files PagesController and UsersController

### PR DESCRIPTION
**Pull request by deveBrice**
mise à jour des fichiers UsersController et PagesController

### CODE REVIEW

File: www/controllers/PagesController.class.php [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <?php
+/*Pages controller*/ 
 class PagesController{
-	
 	public function defaultAction(){
 
 
 		$v = new View("homepage", "back");
 		$v->assign("pseudo","prof");
+		echo 'tata';
```
Comment (alexisljn) [inline www/controllers/PagesController.class.php]: You should display toto instead of tata

File: www/controllers/UsersController.class.php [hunk 2]
```diff
@@ -65,7 +65,8 @@ public function loginAction(){
 
 			if(empty($errors)){
 				$token = md5(substr(uniqid().time(), 4, 10)."mxu(4il");
-                // TODO: connexion
+				// TODO: connexion
+				var_dump($token);
 			}
```
Comment (alexisljn) [inline www/controllers/UsersController.class.php]: Useless line

File: www/controllers/PagesController.class.php (hunk 1 above)
Comment (alexisljn) [inline www/controllers/PagesController.class.php]: You should display toto instead of tata

Comment (alexisljn) [inline www/controllers/PagesController.class.php]: you should display toto instead of tata

Comment (omarreguadi) [inline www/controllers/PagesController.class.php]: why?

File: www/controllers/PagesController.class.php [hunk 3]
```diff
@@ -1,11 +1,12 @@
 <?php
+/*Pages controller*/ 
 class PagesController{
-
```
Comment (omarreguadi) [inline www/controllers/PagesController.class.php]: jfhd

**Pull request merged**